packages/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -33,6 +38,8 @@ @implementation FLTWebViewController {
   int64_t _viewId;
   FlutterMethodChannel* _channel;
   NSString* _currentUrl;
+  // The set of registered JavaScript channel names.
+  NSMutableSet* _javaScriptChannelNames;
```

Comment:
Can use a property and be lazily initialized.

Reply:
What's the benefit over doing it this way?

Reply:
lazy initializing the set can save memory (insignificant tho); In the case of this class, benefit is minimum since the set is used right after class initialization; I'm ok with not lazy initializing here.
Just a thought, if we do move the code `[_javaScriptChannelNames addObjectsFromArray:javaScriptChannelNames];` out of the `initiWithFrame` method, then we can think about lazy initialize the _javaScriptChannelNames to save some memory.

Reply:
We can't move it out of initWithFrame, as a set of JavaScript channels is passed with the creation params... we immediately need this field initialized...